### PR TITLE
Added Thinpool Data & Metadata Utilization Alerts

### PR DESCRIPTION
**Pull request by umangachapagain**
Fixes : gluster/gluster-mixins/issues/4

Signed-off-by: Umanga Chapagain <chapagainumanga@gmail.com>

### CODE REVIEW

**Comment by umangachapagain** [comment]
@shtripat @cloudbehl @anmolsachan

File: alerts/thinpool-utilization.libsonnet [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            alert: 'GlusterThinpoolDataUtilization',
+            expr: |||
+                  100 * gluster_thinpool_data_used_bytes{%(glusterExporterSelector)s} / gluster_thinpool_data_total_bytes{%(glusterExporterSelector)s} > 80
+            ||| % $._config,
+            'for': $._config.thinpoolAlertTriggerTime,
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              message: 'Gluster Thinpool {{ $labels.thinpool }} Data Utilization more than 80%',
```
Comment (shtripat) [inline alerts/thinpool-utilization.libsonnet]: Is there a label like this? I feel the label is `thinpool_name`. You should verify all the labels from https://github.com/gluster/gluster-prometheus/blob/master/docs/metrics.adoc

Comment (umangachapagain) [reply alerts/thinpool-utilization.libsonnet]: changed.

**Comment by shtripat** [comment]
@umangachapagain please rebase

**Comment by umangachapagain** [comment]
Tested. Works fine.

File: alerts/thinpool-utilization.libsonnet [hunk 2]
```diff
@@ -0,0 +1,63 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'thinpool-utilization',
+        rules: [
+          {
+            alert: 'GlusterThinpoolDataUtilization',
+            expr: |||
+                  100 * gluster_thinpool_data_used_bytes{%(glusterExporterSelector)s} / gluster_thinpool_data_total_bytes{%(glusterExporterSelector)s} > 80
```
Comment (anmolsachan) [inline alerts/thinpool-utilization.libsonnet]: Wouldn't `gluster_thinpool_data_used_bytes{%(glusterExporterSelector)s} / gluster_thinpool_data_total_bytes{%(glusterExporterSelector)s} > 0.8 ` work ?  Saves one operation ( 100*  multiplication here) on the metric. Same thing for other alerts.

**Pull request merged**